src/VirtoCommerce.Platform.Web/Security/CustomUserManager.cs

```diff
@@ -22,19 +24,23 @@ public class CustomUserManager : AspNetUserManager<ApplicationUser>
         private readonly IEventPublisher _eventPublisher;
         private readonly IUserPasswordHasher _userPasswordHasher;
```

Comment:
Could you make this IUserPasswordHasher obsolete with notes to use the new password history instead? Previously, IUserPasswordHasher was used for a single purpose only - to save password history.